assignment-2/cmmi.md

```diff
@@ -1,2 +1,136 @@
+| **Level 4**                              |
+| OPP               | Partially | Organizational process performance is covered in a Sprint Retrospective. This retrospective has however little quantitive properties. It is based soley on subjective input of inidividual team members and does not deal with objective measurements. |
+| QPM               | Not at all | Quantitive management of the team's quality and process perfromance is not covered in Scrum.           |
+| **Level 5**                              |
+| CAR               | Not at all | Scrum does not provide a formal process to prevent defects and problems by means of root cause analysis. |
+| OPM               | Partially | The responsibilty of this process is delegated to the Scrum Master who optimizes team performance by removing impediments thus enabling the team to meet its business objectives. There are however, no quantitive properties prescribed. |
+
+## RUP
+
+| Cmmi Process Area | Covered | Rationale  |
+| ------------------| ------- | ---------  |
+| **Level 2**           |
+| CM                | Fully | RUP's configuration and change management fully covers all of this process area's goals            |
+| MA                | Partially | RUP's project management work flow involves an activity for creating a measurement plan although RUP does not exactly specify how this should be communicated to all relevant parties            |
+| PMC               | Fully | The work flow pertaining to project management covers all aspects of monitoring a project against the plan and taking necessary corrective actions              |
+| PP                | Partially | The project management and environment work flows provide adequate support to the project planning with the exception of determining non-software related project attributes such as labor requirements.                 |
+| PPQA              | Fully | The review record in each of RUP's work flow addresses the issue of quality assurance and the creation of the problem resolution plan insures that all quality issues are dealt with.            |
+| REQM              | Fully | The requirements work flow integrates change management along with the capturing and analyzing of requirements.            |
+| SAM               | Not at all | RUP has no elements which dictate interactions with external suppliers it use purely centered around development            |
+| **Level 3**           |
+| DAR               | Not at all | This process area seems to not be covered in RUP in any work flow            |
+| IPM               | Fully | The project management and environment work flows encourages the creation of integrated product plans but are lacking when it comes to the management of dependencies.            |
+| OPD               | Not at all | Due to it being a project methodology and RUP does not prescribe any method for dealing with organizational process assets.             |
```

Comment:
This sentence doesn't flow very well IMO, maybe something like: OPD Doesn't match with RUP at all, because RUP does not prescribe any methods for dealing with organizational process assets.